src/runtime/utils/json.ts

```diff
@@ -11,7 +11,23 @@ export function jsonStringify (value: any) {
  * This function is equivalent to `JSON.parse`, but it also handles RegExp objects.
  */
 export function jsonParse (value: string) {
-  return JSON.parse(value, regExpReviver)
+  return JSON.parse(value, (key, value) => {
```

Comment:
Why not use `decodeURIComponent(body)` in https://github.com/nuxt/content/blob/2ba07760ae04042e2ca20d230a8d8d33ab6a29d1/src/runtime/utils/query.ts#L7

We should not add this only for `_path`. Imagine that users are querying another utf-8 field. We can simply decode whole query simply by adding it in `parseJSONQueryParams` function

Reply:
I'm not sure decode a json object with uri decoder will be a good idea...
also `decodeURIComponent` is for decoding a part of url path (ex.`/to/path`'s `to`) not for the entire url string.

> querying another utf-8 field

Currently the params is encoded to base64 and decode in the server side so I think we don't need to worry about uri encoding in the fly.

I only decode `_path` is because when you use `useRoute().path` it only gets encoded path. so people can directly use it to query the content, or we should decoded it first at the frontend, and remove server side decoding.